pkg/webui/console/views/application-access-edit/index.js

```diff
@@ -18,49 +18,72 @@ import bind from 'autobind-decorator'
 import { Container, Col, Row } from 'react-grid-system'
 import { defineMessages } from 'react-intl'
 import * as Yup from 'yup'
+import { replace } from 'connected-react-router'
 
 import { withBreadcrumb } from '../../../components/breadcrumbs/context'
 import Breadcrumb from '../../../components/breadcrumbs/breadcrumb'
 import sharedMessages from '../../../lib/shared-messages'
-
 import Form from '../../../components/form'
 import Field from '../../../components/field'
 import Button from '../../../components/button'
 import Spinner from '../../../components/spinner'
 import ModalButton from '../../../components/button/modal-button'
 import Message from '../../../lib/components/message'
+import FieldGroup from '../../../components/field/group'
+import diff from '../../../lib/diff'
+import IntlHelmet from '../../../lib/components/intl-helmet'
+import toast from '../../../components/toast'
 
-import { getApplicationsRightsList } from '../../store/actions/applications'
+import { getApplicationApiKey } from '../../store/actions/application'
+import api from '../../api'
 
 import style from './application-access-edit.styl'
 
 const m = defineMessages({
   deleteKey: 'Delete Key',
   modalWarning:
-    'Are you sure you want to delete "{keyName}"? Deleting an application access key cannot be undone!',
+    'Are you sure you want to delete the {keyName} API Key? Deleting an application API Key cannot be undone!',
+  accessEdit: 'Edit {keyName}',
+  updateSuccess: 'Successfully updated API Key',
+  deleteSuccess: 'Successfully deleted API Key',
 })
 
 const validationSchema = Yup.object().shape({
   name: Yup.string()
```

Comment:
There is a mismatch here. In the edit view, the name is required, whereas in the add new view, it is not required. The name should not be required regardless.